inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/projects_knowledge-base.adoc

```diff
@@ -17,7 +17,19 @@ For remote KBs, INCEpTION provides the user with some pre-configured knowledge b
 [.thumb]
 image::kb2.png[align="center"]
 
-In the next step, the user is offered a configurable mapping which relies on a minimal set of IRIs that must be defined for any KB used with the platform. The user can choose one of the pre-configured mapping or provide her own custom defined mapping as shown in the figure below. In the advanced settings, the user can leverage this feature of KB settings when one doesn't want the entire knowledge base to be used and rather choose to identify some specific root concepts. This feature specially helps in case of large knowledge bases such as Wikidata.
+=== Schema mapping
+
+Different types of knowledge base schemata are supported via a configurable mapping mechanism. The user can choose one of the pre-configured mapping or provide her own custom defined mapping as shown in the screenshot below. 
+
+In the advanced settings, the user can leverage this feature of KB settings when one doesn't want the entire knowledge base to be used and rather choose to identify some specific root concepts. This feature specially helps in case of large knowledge bases such as Wikidata.
  
 [.thumb]
-image::kb3.png[align="center"]
\ No newline at end of file
+image::kb3.png[align="center"]
+
+=== Full text search
+
+Full text search support in the knowledge base enables a more efficient access to the knowledge
```

Comment:
Full text search in knowledge bases enables searching for entities by for instance their label. This is a prequesite ...